apps/web/src/pages/integrations/components/ConnectIntegrationForm.tsx

```diff
@@ -350,6 +350,7 @@ const InputWrapper = styled.div`
     margin-bottom: 10px;
     font-size: 14px;
   }
+  padding-right: 30px;
```

Comment:
@nikitagrossman why is the padding here needed?

If any, I would put it on the CenterDiv component instead of on the input wrapppers

Reply:
i did that so the scroll bar would not touch any inputs 
anyway i changed that